app/mailers/audit_mailer.rb

```diff
@@ -0,0 +1,21 @@
+class AuditMailer < ApplicationMailer
+  helper :audits, :layout
+
+  def summary(options = {})
+    raise ::Foreman::Exception.new(N_("Must specify a user with email enabled")) unless (user=User.find(options[:user])) && user.mail_enabled?
+    time     = options[:time] ? "time > \"#{options[:time]}\"" : 'time > yesterday'
+    @query    = "#{options[:query]} and #{time}"
```

Comment:
Which I'm fine with, and is "legal" in scoped_search 
![image](https://cloud.githubusercontent.com/assets/433583/7611727/ef26d9e2-f98e-11e4-988e-b33e0a7e7d62.png)